.github/workflows/scripts/linkerd_deploy.sh

```diff
@@ -8,16 +8,18 @@
 export MESH_NAME='Linkerd'
 export SERVICE_MESH='LINKERD'
 
-curl -fsL https://run.linkerd.io/install | sh
-export PATH=$PATH:/home/runner/.linkerd2/bin
-linkerd version
-linkerd check --pre
-linkerd install | kubectl apply -f -
-linkerd check
+# Check if mesheryctl is present, else install it
+if ! [ -x "$(command -v mesheryctl)" ]; then
+    echo 'mesheryctl is not installed. Installing mesheryctl client... Standby... (Starting Meshery as well...)' >&2
+    curl -L https://meshery.io/install | PLATFORM=kubernetes bash -
```

Comment:
Ditto.

Reply:
Please do suggest and let's get the change committed. `ADAPTERS=linkerd`?